cmd/private-org-sync/main.go

```diff
@@ -365,9 +365,14 @@ func (g gitSyncer) mirror(repoDir string, src, dst location) error {
 
 		if depth == unshallow {
 			logger.Info("Trying to fetch source and destination full history and perform a merge")
-			if err := mergeRemotesAndPush(logger, g.git, repoDir, srcRemote, dst.branch, destUrl.String(), g.confirm, g.gitName, g.gitEmail); err != nil {
+			err, canBeSkippedErr := mergeRemotesAndPush(logger, g.git, repoDir, srcRemote, dst.branch, destUrl.String(), g.confirm, g.gitName, g.gitEmail)
```

Comment:
I think this is totally unnecessary. The `canBeSkippedErr` case can be handled internally in `mergeRemotesAndPush` and the callsite does not need to care, so the change can be much simpler and we don't need to make the interfaces more complicated by having multiple errors with different meanings.